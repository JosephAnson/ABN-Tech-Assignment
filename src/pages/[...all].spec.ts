import { render } from "@testing-library/vue"
import { describe, expect, test } from "vitest"
import All from "./[...all].vue"

describe("All", () => {
  test("should render all page", async () => {
    const { html, getByText } = render(All)
    expect(getByText("We didn't find what you were looking for")).toBeTruthy()
    expect(html()).toMatchSnapshot()
  })
})
