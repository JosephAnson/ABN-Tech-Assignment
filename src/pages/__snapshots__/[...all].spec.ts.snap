// Vitest Snapshot v1

exports[`All > should render all page 1`] = `
"<section class=\\"py-10\\">
  <div class=\\"container max-lg:max-w-full\\">
    <div>We didn't find what you were looking for</div>
  </div>
</section>"
`;
